src/main/java/org.wso2.apim.monetization/impl/StripeMonetizationImpl.java

```diff
@@ -939,6 +1037,8 @@ public boolean publishMonetizationUsageRecords(MonetizationUsagePublishInfo last
             String errorMessage = "Failed to get billing engine data for subscription : " + subscriptionUUID;
             //throw MonetizationException as it will be logged and handled by the caller
             throw new MonetizationException(errorMessage, e);
+        } catch (SQLException e) {
+            e.printStackTrace();
```

Comment:
Why don't we log the exception?

Reply:
fixed in 09f0f2a0ec6b148f13a8092bf53055c9df346ef2